linumpy/psf/psf_estimator.py

```diff
@@ -0,0 +1,141 @@
+import numpy as np
+from linumpy.preproc.xyzcorr import findTissueInterface
+from linumpy.preproc.icorr import confocalPSF, fit_TissueConfocalModel
+
+from scipy.ndimage import binary_dilation, binary_fill_holes, gaussian_filter
+from scipy.stats import zscore
+from skimage.filters import threshold_li
+from skimage.morphology import disk
+
+def extract_psfParametersFromMosaic(
+    vol, f=0.01, nProfiles=10, zr_0=610.0, res=6.5, nIterations=15
```

Comment:
Defaults should probably be set to other values for the new microscope. Very sensitive to the initial zr_0 value.